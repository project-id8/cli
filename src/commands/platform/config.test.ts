import { Command } from "@cliffy/command";
import { snapshotTest } from "@cliffy/testing";
import configCommand from "./config.ts";

// Test help output
await snapshotTest({
  name: "platform config command help",
  meta: import.meta,
  args: ["--help"],
  fn: async () => {
    await new Command()
      .command("platform:config", configCommand)
      .parse();
  },
});

// Test config set command
await snapshotTest({
  name: "platform config set command",
  meta: import.meta,
  args: ["platform:config", "set", "kubeConfig", "/path/to/config"],
  fn: async () => {
    await new Command()
      .command("platform:config", configCommand)
      .parse();
  },
});
