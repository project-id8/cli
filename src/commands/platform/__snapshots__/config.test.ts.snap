export const snapshot = {};

snapshot[`platform config command help 1`] = `
stdout:
"
Usage: COMMAND

Options:

  -h, --help  - Show this help.  

Commands:

  platform:config  - Manage platform configuration

"
stderr:
""
`;

snapshot[`platform config set command 1`] = `
stdout:
"Setting platform config kubeConfig=/path/to/config
"
stderr:
""
`;
